docs/api/python/z3compat/z3compat.rst

```diff
@@ -22,8 +22,3 @@ z3py compatibility Python API
     dt
```

Comment:
Why is BV before arith and array?

Reply:
I put the entries in order based on (a) a rough sense of importance and (b) the order in which I would want someone to read them in. Other options:

1. Keep this idea, but change the order, b/c, e.g., we this arith is more important to a new user.
2. Do the quickstart, followed by core, followed by alpha-order theories, followed by commands/internals/etc.
3. Do everything in alpha-order.

Reply:
I'd favor 2, and I think it matches the "rough sense of importance":
1. Quickstart
2. Core / Boolean
3. Theories
4. Internals

Reply:
Sounds good.